### PR TITLE
pizza structure and pizza size enum

### PR DESCRIPTION
**Pull request by osvegn**
Pull request link with these issues: #41 #40

### CODE REVIEW

File: src/Core/pizza.h [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+/// \file src/Core/pizza.h
+
+#ifndef PIZZA_H_
+    #define PIZZA_H_
+
+    #include <string>
+
+namespace pizza {
+    typedef pizzaSize_e pizzaSize_t;
```
Comment (31Nathan) [inline src/Core/pizza.h:16]: Why the typedef ?

**Review by 31Nathan — CHANGES_REQUESTED** [review]

File: src/Core/pizza.h (hunk 1 above)
Comment (osvegn) [reply src/Core/pizza.h:16]: because

Comment (osvegn) [reply src/Core/pizza.h:16]: Usefull, i don't want to use "enum pizzaSize_e" in the entire project, I prefer "pizzaSize_t"

**Review by 31Nathan — APPROVED** [review]

File: src/Core/Pizza.hpp [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+/// \file src/Core/pizza.h
+
+#ifndef PIZZA_H_
+#define PIZZA_H_
+
+#include <string>
+
+namespace plazza {
+    typedef pizzaSize_e pizzaSize_t;
+    typedef pizza_s pizza_t;
```
Comment (Cavonstavant) [inline src/Core/Pizza.hpp:17]: Not needed

File: src/Core/Pizza.hpp [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    /// \brief Enumeration for available pizza size.
+    enum pizzaSize_e {
```
Comment (Cavonstavant) [inline src/Core/Pizza.hpp:20]: convert to PizzaSize

File: src/Core/Pizza.hpp [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        S = 1,
+        M = 2,
+        L = 4,
+        XL = 8,
+        XXL = 16,
+    };
+
+    /// \brief Structure for pizza.
+    struct pizza_s {
```
Comment (Cavonstavant) [inline src/Core/Pizza.hpp:29]: Convert to Pizza

File: src/Core/Pizza.hpp [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+            /// \brief std::string Name of the pizza.
+            std::string name;
+            /// \brief enum pizzaSize_e Size of the pizza.
```
Comment (Cavonstavant) [inline src/Core/Pizza.hpp:32]: update name

**Review by Cavonstavant — APPROVED** [review]
C structs & enums != C++

**Pull request merged**